TACC/Core-Styles
# fix: font weight black should be 900 not 800

## Overview / Changes

Fix the font-weight mapping for `black`. Make sure "Benton Sans Black" still works.

## Related

- tweaks #305

## Testing

1. Open http://localhost:3000/components/preview/font--cms.
2. Verify Benton Sans Black still appears bolder than "Benton Sans Bold".
2. Verify Benton Sans Black uses the `BentonSans-Black` font.

## UI

| family | weight |
| - | - |
| <img width="920" alt="font family" src="https://github.com/user-attachments/assets/fc876dcc-f80b-49f8-a739-2681800536f9"> | <img width="920" alt="font weight" src="https://github.com/user-attachments/assets/0a41f57c-5ace-4858-829c-e4f039a0aa14"> |
